src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import classNames from 'classnames';
+import _ from 'lodash';
+import { bindClassNames } from '../../util/style-helpers';
+
+const boundClassNames = bindClassNames('Banner');
+
+const {
+	bool,
+	string,
+	oneOfType,
+	oneOf,
+	node,
+	arrayOf,
+	func,
+	element
+} = React.PropTypes;
+
+const defaultIcons = {
+	'success': '<SuccessIcon/>',
+	'danger': '<DangerIcon/>',
+	'info': '<InfoIcon/>',
+	'warning': '<WarningIcon/>',
+	'primary': null,
+	'default': null
+};
+
+/**
+ *
+ * {"categories": ["controls", "banners"]}
+ *
+ * A basic Banner. Any props that are not explicitly called out below will be
+ * passed through to the native `Banner` component.
+ *
+ * Short single line content can be passed in as a simple string. Multi line
+ * messages should be passed wrapped in a `<p>` tag.
+ *
+ * It is valid to use `strong` or `em` within a `Banner` message.
+ */
+const Banner = React.createClass({
+	propTypes: {
+		/**
+		 * pass in a bool to display predefined icon based on `kind`.
+		 */
+		hasIcon: bool,
+		/**
+		 * pass in a icon component for custom icons within banner.
+		 */
+		icon: element,
+		/**
+		 * set this to `true` if you want to have a `x` close icon.
+		 */
+		isCloseable: bool,
+		/**
+		 * set this to `true` if you want to remove rounded corners.
+		 */
+		isFullWidth: bool,
```

Comment:
Does this comment accurately portray the purpose of this prop? If so, the prop should probably be renamed. If not, the comment should probably be more accurate. :)